src/tree/SubscriptionTreeItem.ts

```diff
@@ -38,6 +38,26 @@ export class SubscriptionTreeItem extends SubscriptionTreeItemBase {
         }
     }
 
+    public async pickAppResource(context: IActionContext, options?: PickAppResourceOptions): Promise<AppResourceTreeItem> {
+        await this.getCachedChildren(context);
+        let appResources = GroupTreeItemBase.filterResources(this.cache.appResources);
```

Comment:
If the setting to hide all resources is false, you shouldn't filter the appResources.